wgengine/monitor/monitor_linux.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package monitor
+
+import (
+	"fmt"
+
+	"github.com/mdlayher/netlink"
+	"golang.org/x/sys/unix"
+)
+
+type nlConn struct {
```

Comment:
godoc comment on this type. document why you can't just use a *netlink.Conn?

Reply:
added the comment... just looking to the future where a message might indicate an interface up, down, etc. and each type of event is handled differently.  The message shouldn't be netlink-specific since that'll exclude other connection types.